src/StatusQ/Popups/statusModal/StatusModalHeader.qml

```diff
@@ -42,6 +49,15 @@ Rectangle {
         onEditButtonClicked: statusModalHeader.editButtonClicked()
     }
 
+    MouseArea {
+        anchors.fill: imageWithTitle
+        visible: !!statusModalHeader.popupMenu
+        cursorShape: Qt.PointingHandCursor
+        onClicked: {
+            popupMenuSlot.item.popup(imageWithTitle.x, imageWithTitle.y + imageWithTitle.height + 8)
+        }
+    }
```

Comment:
I think we need to move this into `StatusImageWithTitle` and ensure that it doesn't take the entire with of the header (which the component at the moment does).

This results in the popup opening when it may not be expected.

Also, we should probably discuss with design if there should be a hover indicator (apart form the cursor) when the title is clickable

Reply:
Yes I actually agree with that. currently there is no way to know if the area is clickable except for the mouse cursor

Reply:
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/60327365/132183101-45e88cb4-60f7-4402-9535-980f2dda4eb5.png">


Check comments from carl. We can do this later

Reply:
Gotcha!

Do we still wanna move the mouse area into that other component though and ensure it's not taking the entire width of the header?

Wdyt?

Reply:
I actually kept it that way so that user be easily access it as there is no indicator. Eventually it should be there though